src/ui.h

```diff
@@ -772,12 +769,13 @@ class GraphicsWindow {
     bool ToolbarMouseDown(int x, int y);
     Command toolbarHovered;
 
+    enum    class ShowConstraintMode{SCM_NOSHOW,SCM_SHOW_ALL,SCM_SHOW_DIM,};
     // This sets what gets displayed.
     bool    showWorkplanes;
     bool    showNormals;
     bool    showPoints;
     bool    showConstruction;
-    bool    showConstraints;
+    ShowConstraintMode    showConstraints;
```

Comment:
There are some references to showConstraints where it is still treated as a bool. Perhaps either leave this as a bool and add an extra variable for the display mode, or fix the other uses: including src/textscreens.cpp:171:8